makefont.c

```diff
@@ -198,16 +198,16 @@ int main( int argc, char **argv )
         {
             ++arg;
 
-            if ( 128.0 != texture_width )
-            {
-                fprintf( stderr, "Multiple --texture parameters.\n" );
-                print_help();
-                exit( 1 );
-            }
+            // if ( 128.0 != texture_width )
```

Comment:
Please don't comment out sections. Delete them if they are not needed.

Reply:
Totally agree, not sure why I did that (was probably while I was trying things out).

Reply:
Done.